apps/backend/src/database/migrations/1692293473030-migration.ts

```diff
@@ -0,0 +1,23 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migration1692293473030 implements MigrationInterface {
+  name = 'migration1692293473030';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
Is this migration going to run okay on test? Since we already data in this column?